src/generateTypings.ts

```diff
@@ -4,36 +4,51 @@ import * as path from 'path';
 import * as chalk from 'chalk';
 import * as mkdirp from 'mkdirp';
 import * as globby from 'globby';
+const stylus: any = require('stylus');
 const DtsCreator: any = require('typed-css-modules');
 const read: any = require('read-file-stdin');
 
 export interface TypingsArgs extends Argv {
 	in: string;
 	out: string;
+	stylus: boolean;
 }
 
-function readFile(fileName: string) {
+function readFile(fileName: string): Promise<string> {
 	return new Promise((resolve, reject) => {
-		read(fileName, (err: Error, buffer: Buffer) => {
+		read(fileName, (err: Error, content: string) => {
 			if (err) {
 				reject(err);
 				return;
 			}
 
-			resolve(buffer);
+			resolve(content.toString());
 		});
 	});
 }
 
-function write(name: string, content: { formatted: string }) {
+function compileStylus(content: string): Promise<string> {
```

Comment:
Would be good to walk through this with you to see how the process fits together, would this not be better in a `compileStylus` file? Doesn't seem like a direct concern of `generateTypings`. Though that's just my opinion.